docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,165 @@
+To create tests for this page, create the *my-project* directory anywhere on your computer and install TestCafe into it. You can do this from a command line.
+
+```bash
+mkdir my-project
+cd my-project
+npm install --save-dev testcafe
+```
+
+## Writing Test
+
+You can organize your tests into any directory on your computer. It is a good practice to create the *test* directory as a subdirectory of the website project you are going to test and place tests there.
+
+```bash
+cd my-project
+mkdir test
+```
+
+To create a test, go to the *test* directory, create a new .js file and add your code to it.
+
+### General Code Structure
+
+TestCafe organizes tests into fixtures within a .js file. This file can contain several fixtures as well as each fixture can contain multiple tests.
+
+The following snippet illustrates a basic structure of the .js file containing tests:
+
+```js
+fixture `Fixture 1 name`
+
+test('Test 1 name', async t => {
+    //Test 1 code
+});
+
+test('Test 2 name', async t => {
+    //Test 2 code
+});
+
+fixture `Fixture 2 name`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('Test 1 name', async t => {
+    //Test 1 code
+});
+
+test('Test 2 name', async t => {
+    //Test 2 code
+});
+```
+
+You declare fixtures by using the `fixture` function. You can optionally specify a start page URL for a fixture by using the `page` function. This means that all tests within this fixture will target the same page.
+
+Test code is placed within the `test` function.
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test for the [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example) sample page.
+The test types a developer name into a text editor and then clicks the Submit button.
+You can try running this test against the sample page by yourself.
+
+```js
+fixture `Getting Started - Test 1`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+All test actions are implemented as asynchronous functions of the test controller object `t`. This object is used to access test run API.
+To wait for actions to be complete, use the `await` function when calling these actions or action chains.
+
+### Observing Page State
+
+TestCafe allows you to perform custom actions on a page element and observe the element's state.
+For this purpose, it provides the `Hybrid` function.
+You wrap your code into this function, and it is executed on the client side, thus giving you direct access to DOM elements.
```

Comment:
>  perform custom actions on a page element

this is bad practice.

> You wrap your code into this function

You don't "wrap" anything. You create function with code which will be executed on client.